Remotion/Web/Core/UI/Controls/WebTab.cs

```diff
@@ -91,7 +91,7 @@ protected virtual void PreRender()
   }
 
   /// <summary> Sets this tab's <see cref="WebTabStrip"/>. </summary>
-  protected internal virtual void SetTabStrip (WebTabStrip tabStrip)
+  protected internal virtual void SetTabStrip (WebTabStrip? tabStrip)
```

Comment:
```suggestion
  protected internal virtual void SetTabStrip (WebTabStrip tabStrip)
```
I think....

Reply:
There are some places where nullable values or null are passed in, e.g. the follwing.

https://github.com/re-motion/Framework/blob/b05cd9007c55e32fd8656ba99cfc41392f74fc88/Remotion/Web/Core/UI/Controls/WebTabCollection.cs#L100